### PR TITLE
Fix submitting mails with proposals

### PR DESCRIPTION
**Pull request by deiferni**
- Add missing workflow for `SubmittedProposal`, allow adding `ftw.mail.mail`
- Add initial version to mails, allow tracking as `SubmittedDocument`

### CODE REVIEW

**Comment by deiferni** [comment]
@phgross

File: opengever/meeting/profiles/default/workflows/opengever_submitted_proposal_workflow/definition.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<dc-workflow workflow_id="opengever_submitted_proposal_workflow" title="SubmittedProposal Workflow" description="" state_variable="review_state" initial_state="proposal-state-active" manager_bypass="False">
+ <permission>ftw.mail: Add Mail</permission>
+ <state state_id="proposal-state-active" title="proposal-state-active">
+  <permission-map name="ftw.mail: Add Mail" acquired="True">
+   <permission-role>Administrator</permission-role>
+   <permission-role>Contributor</permission-role>
+   <permission-role>Manager</permission-role>
+   <permission-role>Owner</permission-role>
```
Comment (phgross) [inline opengever/meeting/profiles/default/workflows/opengever_submitted_proposal_workflow/definition.xml]: :scissors:

File: opengever/meeting/upgrades/profiles/4300/workflows/opengever_submitted_proposal_workflow/definition.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<dc-workflow workflow_id="opengever_submitted_proposal_workflow" title="SubmittedProposal Workflow" description="" state_variable="review_state" initial_state="proposal-state-active" manager_bypass="False">
+ <permission>ftw.mail: Add Mail</permission>
+ <state state_id="proposal-state-active" title="proposal-state-active">
+  <permission-map name="ftw.mail: Add Mail" acquired="True">
+   <permission-role>Administrator</permission-role>
+   <permission-role>Contributor</permission-role>
+   <permission-role>Manager</permission-role>
+   <permission-role>Owner</permission-role>
```
Comment (phgross) [inline opengever/meeting/upgrades/profiles/4300/workflows/opengever_submitted_proposal_workflow/definition.xml]: :scissors:

**Comment by phgross** [comment]
@deiferni Looks good. Just a small issue in the permission-map

**Comment by deiferni** [comment]
@phgross updated

**Pull request merged**